.github/workflows/trigger-semgrep-comparison-argo.yaml

```diff
@@ -1,4 +1,7 @@
-name: trigger-semgrep-comparison-argo-workflow
+# This workflow triggers the `semgrep-comparison` Argo Workflow, which will
+# create a GitHub Check on this pull request (or specified branch if running manually.
```

Comment:
missing ), and I don't see any on: pull_request, so I don't think this run on a pull_request.

Reply:
This workflow will be called from the `tests` workflow, which runs on pull requests, but that will come in a forthcoming PR.